server/application.js

```diff
@@ -1,55 +1,77 @@
 const express = require('express');
+// const morgan = require('morgan');
 const path = require('path');
-const db = require('../database/index');
+const db = require('../database/config');
 
 const app = express();
 
+// app.use(morgan('dev', {
+//   skip: (req, res) => res.statusCode < 400,
+//   stream: process.stderr,
+// }));
+
+// app.use(morgan('dev', {
+//   skip: (req, res) => res.statusCode >= 400,
+//   stream: process.stdout,
+// }));
+
```

Comment:
Just a quick note to tidy up the 'morgan' cruft here if it's not going to be used. Chunks of commented-out code section can be confusing for maintainers because it's tough sometimes to tell if it was just a temporary removal, or the feature is dead and it should have been removed but was forgotten, or perhaps it _is_ supposed to here for use in testing or something.

Reply:
Oops, forgot to re-comment it in. Was using it for logging when testing server but had turned it off when stress-testing. Re-commented in the most recent push.